### ISSUE TITLE
Aufnahme auf asksinpp.de

### ISSUE THREAD

**Issue by jp112sdl** [netprog2019/Homematic-HomeBrew-Komponenten#1]
Hi,

möchtest du deinen verwendeten Model-ID-Bereich (`0xFB..`) auch mit auf _asksinpp.de_ vermerkt haben?
https://asksinpp.de/rftypes/HomeBrew.html

VG,
Jérôme

**Comment by netprog2019** [comment]
Servus Jérôme,

ich bin noch gaNZ am Anfang mit der ganzen Materie. Ich habe mich aber parallel auch mit github beschäftigt und
schon etwas Dokumentation gebaut.
Ein paar eigene Sachen gehen auch schon, wie z.Bsp. der Garagensensor, incl. XML auf der RaspberryMatic.
Allerdings werden im Moment bei Temperatur, Luftfeuchte und Luftdruck nur Fake-Werte übertragen. ShutterContact und SmokeDetektor mit MQ-135 funktionieren.
Wie gesagt, ich bin noch am Aufbau des ganzen.
Ich würde mich freuen, wenn Du meinen Device-Model-Bereich (0xFB..0FC) mit auf  asksinpp.de aufnehmen würdest, 
allerdings weiß ich nicht, ob es noch etwas verfrüht dafür ist. 

Viele Grüße aus Pirna
Maik

**Comment by jp112sdl** [comment]
> Device-Model-Bereich (0xFB..0FC)

**Comment by netprog2019** [comment]
> Du meinst damit sicher den gesamten Bereich `0xFB00-0xFBFF`
Ja, sorry - so meinte ich das. Du bist ja schnell gewesen - https://asksinpp.de/rftypes/HomeBrew.html#bereichsubersicht

> Ist dann auf jeden Fall erstmal "reserviert"
Danke dafür.

> Ja, hab schon deine unzähligen Commits zur README gesehen. :)
Hm, da ich leider nicht die Englischkenntnisse besitze, die ich bräuchte, ist das ein etwas Steiniger Weg für mich.
Somit muss ich mal schauen, ob ich die ganzen Commits löschen kann und die Datei erstmal lokal bearbeite. 
Also "learning by doing"

LG Maik

**Comment by jp112sdl** [comment]
Du kannst ja vor dem "Commit" erstmal auf "Preview" klicken und dir anschauen, wie die README nach deiner Änderungen aussehen wird.
